### PR TITLE
Fix for issue #25 - Creating welcome page

### PR DESCRIPTION
**Pull request by deepanjali19**
I have made the welcome page for the website.

The only issue left is that website does not directly go to the GitHub Dashboard application if user is already logged in.
The user will first land on the welcome page, after clicking the "Login to Github" button, user is redirected to the dashboard.

### CODE REVIEW

File: src/Welcome.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                            <a className="navbar-brand" href="#">
+                                <span id="dashboard">GitHub-Dashboard</span>
+                            </a>
+                        </div>
+                    </div>
+                </nav>
+
+                <div className="container">
+                    <div className="row">
+                        <a href="#" className="navbar-left"><img src={WelcomeImage} height="400" alt="Welcome" /></a>
```
Comment (jkmdev) [inline src/Welcome.js]: I think the link should be removed from this image, so that line 26 is just:

`Welcome`

**Comment by jkmdev** [comment]
Honestly I think we could just make a separate issue for the 'Route to home if you're logged in' feature. People can still do development work with the welcome page as it is.

There have been some recent changes made to master, they'll have to be merged into this branch before I can merge it. After that it should be good to go!

File: src/Welcome.js (hunk 1 above)
Comment (deepanjali19) [reply src/Welcome.js]: Working on it!

**Comment by deepanjali19** [comment]
@jkmdev ok I can create a new issue for the rest of the development. I will update this branch and push the final code today.

File: src/Welcome.js (hunk 1 above)
Comment (jkmdev) [reply src/Welcome.js]: Ayyyy, alright, missed that, makes sense

**Pull request merged**